models/articles.model.js

```diff
@@ -83,4 +83,39 @@ exports.selectCommentsByArticleId = async (targetArticleId) => {
             return comments;
         }
     }
+}
+
+exports.addCommentByArticleId = async (targetArticleId, incomingComment) => {
+    const votes = 0;
+    const created_at = '2020-06-20T07:24:00.000Z';
```

Comment:
the seed file has psql within it that creates a 'created_at' timestamp by default (line 37 -  `created_at TIMESTAMP DEFAULT NOW(),`) so you don't actually need to send a created_at value when you post a new comment 

Also votes has a default value in the seed PSQL too so you don't need to send that either